lib/commands/dialog/index.js

```diff
@@ -89,7 +89,7 @@ class DialogCommand extends AbstractCommand {
                     new ResourcesConfig(path.join(process.cwd(), CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG));
                     skillId = ResourcesConfig.getInstance().getSkillId(profile);
                 } catch (err) {
-                    throw 'Failed to read project resource file.';
+                    throw 'Failed to read project resource file. Please run the command within a ASK-CLI project.';
```

Comment:
should throw CliError class error or subclass of it. In the test you can verify that the error throw is instance of CliError class

Reply:
acknowledged.